packages/FormElement/src/FormElement.js

```diff
@@ -147,15 +165,17 @@ function FormElement(props) {
       <div css={isInline ? inlineContainerStyles : null}>
         {renderInstructions()}
         {extractedChildren.children.map(child => {
-          const extendedProps = isNativeElement(child)
-            ? {
+          let extendedProps = {};
+          if (!inputFound) {
```

Comment:
Currently formElement only really supports a single input component to `setErrors` etc on it. We really only want to  support adding these extendedProps to one supported component.